_includes/releases/v22.1/v22.1.0-alpha.4.md

```diff
@@ -0,0 +1,97 @@
+- Added support for super region functionality. Super regions allow the user to define a set of regions on the database such that any `REGIONAL BY TABLE` based in the super region or any `REGIONAL BY ROW` partition in the super region will have all their replicas in regions within the super region. The primary use is for data domiciling. Super regions are an experimental feature and are gated behind the session variable: `enable_super_regions`. The [cluster setting](../v22.1/cluster-settings.html) `sql.defaults.super_regions.enabled` is used to enable super regions (default: `false`). [#76620][#76620]
+
+<h3 id="v22-1-0-alpha-4-operational-changes">Operational changes</h3>
+
+- Added the `server.shutdown.connection_wait` [cluster setting](../v22.1/cluster-settings.html) to the [draining process](../v22.1/node-shutdown.html#node-shutdown-sequence) configuration. This adds a draining phase where the server waits for SQL connections to be closed, and once all SQL connections are closed before timeout, the server proceeds to the next draining phase. This provides a workaround when customers encounter intermittent blips and failed requests when performing operations that are related to restarting nodes. [#72991][#72991]
+- The [cluster settings](../v22.1/cluster-settings.html) `admission.kv.tenant_weights.enabled` and `admission.kv.stores.tenant_weights.enabled` can now be used to enable tenant weights in multi-tenant storage servers (Defaults: `false`). Tenant weights are based on the number of ranges for each tenant, and allow for weighted fair sharing. [#77575][#77575]
+
+<h3 id="v22-1-0-alpha-4-command-line-changes">Command-line changes</h3>
+
+- The `cockroach debug tsdump` command now allows viewing time-series data even in case of node failures by being re-run with the import filename set to `-`. [#77247][#77247]
+
+<h3 id="v22-1-0-alpha-4-db-console-changes">DB Console changes</h3>
+
+- Added an alert banner on the [**Cluster Overview** page](../v22.1/ui-cluster-overview-page.html) that indicates when more than one node version is detected on the cluster. The alert lists the node versions detected and how many nodes are on each version. This provides more visibility into the progress of a cluster upgrade. [#76932][#76932]
+- The **Compactions/Flushes** graph on the [Storage dashboard](../v22.1/ui-storage-dashboard.html) now shows bytes written by these operations, and has been split into separate per-node graphs. [#77558][#77558]
+- The [**Explain Plan**](../v22.1/ui-statements-page.html#explain-plan) section of the [**Statement Details** page](../v22.1/ui-statements-page.html#statement-details-page) now shows statistics for all the plans executed by the selected statement on the selected period. [#77632][#77632]
+- Active operations can now be inspected in a new **Active operations** page linked from the [**Advanced Debug** page](../v22.1/ui-debug-pages.html). This facilitates viewing active traces and taking snapshots. [#77712][#77712]
```

Comment:
Nit: Should the bold "Active operations" be capped on the "Operations", since it's a new page?